### DISCUSSION TITLE
Random Chats About Theming

### DISCUSSION THREAD

**Discussion by CreativeCodeCat** [themix-project/themix-gui#399] [Ideas]
New space sop we don't fill up [397](https://github.com/themix-project/oomox/issues/397)™

**Comment by CreativeCodeCat** [comment]
if @gusbemacbe would like to have a play about with [Sardi](https://github.com/erikdubois/Sardi) and [Surfn](https://github.com/erikdubois/Surfn) icons, there are videos of erik using a script to change the colors but i cant seem to find the script in any of his repos but i am in contact with him via telegram

**Comment by CreativeCodeCat** [comment]
below is part of the script that i can find that he uses

```
# Folder top smaller portion
find scalable/places -name "*.svg" -type f -exec sed -i 's/fill="#'$oldcolourtop'"/fill="#'$newcolourtop'"/g' {}  \;
find scalable/places -name "*.svg" -type f -exec sed -i 's/fill:#'$oldcolourtop'/fill:#'$newcolourtop'/g' {}  \;

# Folder body portion 
find scalable/places -name "*.svg" -type f -exec sed -i 's/fill="#'$oldcolourbody'"/fill="#'$newcolourbody'"/g' {}  \;
find scalable/places -name "*.svg" -type f -exec sed -i 's/fill:#'$oldcolourbody'/fill:#'$newcolourbody'/g' {}  \;

# Icons on the folders
find scalable/places -name "*.svg" -type f -exec sed -i 's/fill="#'$oldcolouricons'"/fill="#'$newcolouricons'"/g' {}  \;
find scalable/places -name "*.svg" -type f -exec sed -i 's/fill:#'$oldcolouricons'/fill:#'$newcolouricons'/g' {}  \;

# Changing the stroke colour
find scalable/places -name "*.svg" -type f -exec sed -i 's/stroke:#'$oldcolouricons'/stroke:#'$newcolouricons'/g' {}  \;
find scalable/places -name "*.svg" -type f -exec sed -i 's/stroke="#'$oldcolouricons'"/stroke="#'$newcolouricons'"/g' {}  \
```

**Comment by CreativeCodeCat** [comment]
Might look into this tomorrow as I have some spare time on my hands then and I hope to get it to work :)

**Reply by actionless** [reply]
nice, feel free to ask me anything - usually on the weekend i'm busy recording some audio or sleeping, but i'm checking the mail from time to time to answer some quick questions

**Reply by CreativeCodeCat** [reply]
not really had the time to work on thjis as been working on my own btrfs snapshot tool but i will set aside some time over the next few weeks to get this worked on :)

**Reply by actionless** [reply]
i saw in notifications, you had some problem with getting theme values from the gtk theme, did you already found them?

**Reply by CreativeCodeCat** [reply]
ya mate i got it working i had to add accent to the manual mode i also made a pull request for it and for a few extra things to make qt5 and qt6 work with other themes

**Reply by actionless** [reply]
for some reason i'm gettin emails only for replies, but not for the new topics/PRs

**Reply by actionless** [reply]
and about qt{5,6}ct themes - current implementation is a bit silly hack - and would be better to use your plugin in the future

**Reply by CreativeCodeCat** [reply]
ok ill make a template for this in my plugin :) as long as the correct colors are there

**Comment by actionless** [comment]
sup @The-Repo-Club i've recently made a new release - please make sure your plugin is still working :)

**Comment by CreativeCodeCat** [comment]
this may need fixing? oh thats my end but ill fix it
![image](https://user-images.githubusercontent.com/9284733/136602756-5b64bfa7-1e70-402c-bad4-68cf45291276.png)

**Reply by actionless** [reply]
are you using the same hotkey in your plugin as in xresources?

**Reply by CreativeCodeCat** [reply]
ya i fixed it now :)

**Reply by actionless** [reply]
btw just added new feature a minute ago - now Shortcuts window also shows custom shortcuts defined by plugins :)
